README.md

```diff
@@ -105,3 +105,34 @@ secrets**, so they are attached to the correct worker:
 ```bash
 npx wrangler@latest secret put MY_SECRET --name $(git rev-parse --short HEAD)
 ```
+
+## End-to-End Testing
+
+### Installation
+
+**Install Grafana k6**
+
+- **macOS**
+
+  ```bash
+  brew install k6
+  ```
+
+- **Debian/Ubuntu**
+
+  ```bash
+  sudo gpg -k
+  sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
+  echo "deb [signed-by=/usr/share/keyrings/k6-archive-keyring.gpg] https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
+  sudo apt-get update
+  sudo apt-get install k6
+  ```
+
+For detailed installation instructions, see the
+[official grafana k6 installation guide](https://grafana.com/docs/k6/latest/set-up/install-k6/).
```

Comment:
Let's omit these steps; the instructions might get outdated, especially the keys. Instead, let's just add a section `dependencies` and link to `k6`. If the reader is not able to follow the link and figure out how to install `k6`, they should follow a different career path. :rofl: